data/admin/ingredient.py

```diff
@@ -49,3 +49,6 @@ class IngredientAdmin(ElementAdminWithChangeReason):
         "siccrf_private_comments",
     )
     search_fields = ["id", "name"]
+
+    def has_linked_substances(self, obj):
+        return "Oui" if obj.substances.count() else "Non"
```

Comment:
Si on n'est pas intéressé par le nombre `exists` est plus adapté
```suggestion
        return "Oui" if obj.substances.exists() else "Non"
```